app/views/organisations/_header.html.erb

```diff
@@ -1,5 +1,62 @@
 <% if @organisation.is_no_10? %>
-  <h1 class="organisation__no10-banner organisation__margin-bottom"><%= @header.org.title %></h1>
+  <div class="govuk-width-container">
+    
+    <div class="organisation__no10-header organisation__margin-bottom">
+      <%# To do: add asset here and style and name this container as needed %>
+      <span class="govuk-heading-xl govuk-!-text-align-centre">
+      10
+      </span>
+      
+      <div class="organisation__no10-header__banner govuk-!-margin-bottom-6">
+
+        <%= content_tag(:picture) do %>
+          <%= content_tag(:source, nil, { media: "(min-width: 641px)", srcset: "#{image_path('no10header-desktop.png')}" }) %>
+          <%= content_tag(:img, nil, { srcset: "#{image_path('no10header.png')}"}) %>
+        <% end %>
+
+        <div class="organisation__no10-header__banner__content">
+          <h1 class="organisation__no10-header__banner__content__heading govuk-heading-l govuk-!-margin-0">
```

Comment:
You can also use the heading component here. The heading component has an 'inverse' option so you can make the text white.